Tests/SwiftCliCoreTests/NestedArrayBuilderTest.swift

```diff
@@ -16,3 +16,23 @@ class NestedArrayMakerTest: XCTestCase {
     }
 
 }
+
+class MultipleStitchExpanderTest: XCTestCase {
+
+
+    func testExpandValidMultipleStitch() throws {
+        let result = try NestedArrayBuilder().expandRow(row: ["k4", "p1"])
+        expect(result).to(equal(["k1", "k1", "k1", "k1", "p1"]))
+    }
+
+    func testInvalidZeroCountStitch() throws {
+
+        expect{ try NestedArrayBuilder().expandRow(row: ["k0", "p1"]) }.to(throwError(InputError.invalidStitchNumber(rowNumber: nil, invalidStitch: "k0", validStitchType: "k", invalidStitchNumber: "0")))
+    }
+
+    func testInvalidWordCountStitch() throws {
+
+        expect{ try NestedArrayBuilder().expandRow(row: ["kx", "py"]) }.to(throwError(InputError.invalidStitchNumber(rowNumber: nil, invalidStitch: "kx", validStitchType: "k", invalidStitchNumber: "x")))
+    }
```

Comment:
You can delete the extra lines above the expect.  Also, maybe play with leaving the code as is, but seeing if swift will allow you to break it into separate lines for readibility.  Like

`var obj = {name: "Tony", age: "Go away"}`
vs
```
var obj = {
  name: "Tony", 
  age: "Go away"
}
```

Reply:
That is more readable!